src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectory.Stream.Tests.cs

```diff
@@ -97,9 +97,22 @@ public void Extract_LinkEntry_TargetOutsideDirectory(TarEntryType entryType)
         [ConditionalFact(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
         public void Extract_SymbolicLinkEntry_TargetInsideDirectory() => Extract_LinkEntry_TargetInsideDirectory_Internal(TarEntryType.SymbolicLink);
 
-        [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/68360", TestPlatforms.Android | TestPlatforms.iOS | TestPlatforms.tvOS)]
+        private static bool IsUnixAndDoesNotNeedElevationForHardLinkCreation =>
+            !PlatformDetection.IsWindows &&
+            !PlatformDetection.IsAndroid && !PlatformDetection.IsiOS && !PlatformDetection.IstvOS;
+
+        [ConditionalFact(nameof(IsUnixAndDoesNotNeedElevationForHardLinkCreation))]
         public void Extract_HardLinkEntry_TargetInsideDirectory() => Extract_LinkEntry_TargetInsideDirectory_Internal(TarEntryType.HardLink);
+        
+        private static bool IsUnixAndNeedsElevationForHardLinkCreation =>
+            RemoteExecutor.IsSupported && PlatformDetection.IsSuperUser &&
+            (PlatformDetection.IsAndroid || PlatformDetection.IsiOS || PlatformDetection.IstvOS);
```

Comment:
Do we ever run as su on these platforms? I didn't even know you could do that on a mobile OS.

Reply:
Also looks like `RemoteExecutor.IsSupported` isn't supported on these platforms either:

https://github.com/dotnet/arcade/blob/5fdd5b6c08c6c39edf12001400a11786c03b3e35/src/Microsoft.DotNet.RemoteExecutor/src/RemoteExecutor.cs#L98-L107

Reply:
@akoeplinger confirmed we do not.

I need to find out if hardlinks can be created on mobile platforms. If not, we will have to skip them.

Reply:
Not possible: https://code.google.com/archive/p/android-developer-preview/issues/3150 (thanks for the link, @akoeplinger )